### PR TITLE
DAH-1807 fix: remove logs, listing_image_service, refactor

### PR DESCRIPTION
**Pull request by cade-exygy**
DAH-1807 

Refactor Listing Image Caching to no longer use ListingImageService (and only use MultipleListingImageService).

### CODE REVIEW

File: app/services/cache_service.rb [hunk 1]
```diff
@@ -37,11 +32,13 @@ def cache_only_updated_listings
         l['Id'] == fresh_listing['Id']
       end
 
-      cache_single_listing(fresh_listing) unless listing_unchanged?(prev_cached_listing,
-                                                                    fresh_listing) && listing_images_unchanged?(
-                                                                      prev_cached_listing, fresh_listing
-                                                                    )
-      cache_listing_images && process_listing_images(fresh_listing)
+      ## todo: remove listing_unchanged because it doesn't work?
```
Comment (tallulahkay) [inline app/services/cache_service.rb:35]: @cade-exygy I found that listing_unchanged _does_ work - do you want to pair and compare what we found?

File: config/environments/development.rb [hunk 2]
```diff
@@ -57,7 +57,7 @@
     }
 
     # These numbers are from the recommended dalli config: https://devcenter.heroku.com/articles/memcachier#ruby
-    config.cache_store = :dalli_store, nil, {
+    config.cache_store = :mem_cache_store, nil, {
```
Comment (tallulahkay) [inline config/environments/development.rb:60]: this is an update to make caching work locally

**Comment by tallulahkay** [comment]
@cade-exygy I think this is finally ready for a review

I included review instructions on [this ticket](https://sfgovdt.jira.com/browse/DAH-1812)

**Comment by jimlin-sfgov** [comment]
issue: I wasn't able to get a newly added listing image to show up on the Listing details page.

steps to replicate:
1. run `bundle exec rails cache:prefetch_daily` as the review instructions say
2. add image to selected listing with this url: `https://sfhousing--full.sandbox.lightning.force.com/lightning/r/Listing__c/a0W4U00000KnXGsUAN/related/Listing_Images__r/view`
3. run `bundle exec rails cache:prefetch` in the attached bash terminal
4. fail to see `Calling cache_single_listing a0W4U00000KnXGsUAN` in the attached bash terminal
5. newly added image not showing on Listing Details page (should it?)

**Comment by tallulahkay** [comment]
@jimlin-sfgov thank you for reviewing! let's pair later to look at this? It should have worked, so maybe I missed something in the review instructions

**Review by jimlin-sfgov — APPROVED** [review]
Manual testing passes, after pairing and tweaking the review instructions. Great job!

**Pull request merged**